### ISSUE TITLE
Crashes during orb run

### ISSUE THREAD

**Issue by Sandman25DCSS** [jeremygurr/dcssca#702]
There were several crashes during orb run on Zot  6 and Zot 5. Unfortunately it looks like chash files have not generated.

By the way, I really like new orb run, though probably it should disable scrolls of magic mapping also.

**Comment by jeremygurr** [comment]
Cool I'm glad you like it. Now we just got to get those crashes worked out. What was happening just before the crash? Any idea what could have triggered it? Oh it was when you were trying to cast tornado? Well someone was crashing it with tornado. Hmm... I might have an idea.

Ok I fixed the tornado bug. If that was your issue then it should be good now. I think you already finished that game though, huh?
